scripts/genbindings.sh

```diff
@@ -28,6 +28,24 @@ function checkInstalledLocally() {
   fi
 }
 
+function compileAndMoveTo() {
+  if [ ! -d "$1" ] || [ ! -d "$2" ]; then
+    echo ERROR: No valid source or destination specified
+    echo compileAndMoveTo [source] [destination]
+    exit 1
+  fi
+
+  CWD="${PWD}"
+  cd "${1}"
+
+  mkdir -p "${2}"/contracts
+  mkdir -p "${2}"/contracts/abi
+  npx hardhat compile && cp -r ./typechain/* "${2}"/contracts/ \
```

Comment:
We use `yarn` everywhere, so I propose
```suggestion
  yarn hardhat compile && cp -r ./typechain/* "${2}"/contracts/ \
```

Reply:
Why copy it instead of moving it?
```suggestion
  npx hardhat compile && mv ./typechain/* "${2}"/contracts/ \
```